### ISSUE TITLE
"Build it yourself" documentation does not work

### ISSUE THREAD

**Issue by rajb245** [Tzahi12345/YoutubeDL-Material#728]
I do a fresh `git clone`, `npm install` in the root directory of this project, `npm install` in the backend directory, modify `appdata/default.json` to be how I want it, and then try `npm build` in the root directory of the project. I get the following warning:

> npm WARN build `npm build` called with no arguments. Did you mean to `npm run-script build`?

Then I go look in `backend` for a `public` directory, but none was generated. I know my way around building software but have zero experience with web stuff / node. I saw there is indeed a script called build in package.json, so I tried to run the suggested command that the warning output. But then, I get the following error
```
> youtube-dl-material@4.3.0 prebuild /home/raj/youtubedl-material-git
> node src/postbuild.mjs

/home/raj/youtubedl-material-git/src/postbuild.mjs:1
import fs from 'fs-extra';
       ^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
```

But `package.json` has the dependency on fs-extra listed there and I go look in node_modules of the top level directory and there is a fs_extra directory in there.

My request is for the docs to be updated for how to build from scratch. System specifics:
npm --version
6.14.4
Ubuntu 22.04

**Comment by martadinata666** [comment]
likely typo on build, instead `npm build` use `npm run build`
1. Clone repo
2. Goto root  project `npm install
3. Goto backend dir `npm install`
4. Return to roo project `npm run build`
5. Install global deps `npm -g install pm2`
6. Goto backend dir
7. Modifiy appdata/default.json
8. run `npm start`
9. Ytdl will start in background.

**Comment by Tzahi12345** [comment]
README was fixed with #800, will close now. Sorry for the confusion! (it was supposed to be `npm run build`)
